ionoscloud/resource_lan.go

```diff
@@ -89,6 +100,15 @@ func resourceLanCreate(ctx context.Context, d *schema.ResourceData, meta interfa
 		request.Properties.Pcc = &pccID
 	}
 
+	if d.HasChange("ipv6_cidr_block") {
```

Comment:
why use haschange in a post req?

Reply:
fixed to simply use Get and explicitly set nil value